src/napari_cryoet_data_portal/_open_widget.py

```diff
@@ -133,6 +143,10 @@ def _loadTomogram(
         image_attrs["scale"] = tuple(
             resolution.scale * s for s in image_attrs["scale"]
         )
+        image_translate = image_attrs.get("translate", (0,) * len(image_attrs["scale"]))
```

Comment:
@kephale : I believe the tomograms don't have translations at the moment, but this assumes they could to a bit of future-proofing. What do you think?

Reply:
I think this needs to be sensitive to the downsampling method (average over bin, top-left corner, etc.). Since we're processing these zarrs, I guess we can make an assumption about the methd...

Reply:
Yeah, I'm fine assuming the method here for now.

OME-NGFF does have [a place for downsampling method](https://ngff.openmicroscopy.org/latest/#multiscale-md), but looks like we don't currently fill it.